### DISCUSSION TITLE
MkDocs site available to other computer on local network

### DISCUSSION THREAD

**Discussion by brianjoubert** [mkdocs/mkdocs#3059] [Q&A] [answered]
Hello,

I was wondering if it was possible to make the MkDocs site available to other users on a local network. It is working on my computer but nobody on my local network can access it. Is it a limitation of MkDocs or my Network administrator?

Best

**Comment by pawamoy** [comment, marked as answer]
Try it to serve it on 0.0.0.0. Then share your local IP to your peers: they will be able to access your site on that IP, on the port you serve with.

Local IPs look like 192.168.x.x.

**Comment by brianjoubert** [comment]
Thank you very much. It works like a charm!
